src/main/java/org/apache/sling/servlets/post/impl/operations/AbstractPostOperation.java

```diff
@@ -313,6 +311,102 @@ protected String getResourcePath(SlingHttpServletRequest request) {
         return request.getResource().getPath();
     }
 
+
+    /**
+     * Orders the given node according to the specified command. The following
+     * syntax is supported: &lt;xmp&gt; | first | before all child nodes | before A |
+     * before child node A | after A | after child node A | last | after all
+     * nodes | N | at a specific position, N being an integer &lt;/xmp&gt;
+     *
+     * @param request The http request
+     * @param resource the resource to order
+     * @param changes the list of modifications
+     * @throws PersistenceException in case the operation is not successful 
+     */
+    protected void orderNode(final SlingHttpServletRequest request,
```

Comment:
I think that ```orderResource``` is more appropriate now.

Reply:
Done in https://github.com/apache/sling-org-apache-sling-servlets-post/pull/16/commits/6d99e46c820f63f5a623d0bdcae514f182a987bd